.github/workflows/release.yml

```diff
@@ -10,36 +10,19 @@ on:
         # force ourselves to always create them locally.
         description: ⚠️ This workflow can only automatically release patch versions, or Babel 8 pre-releases
         required: true
-        default: patch
+        default: prerelease
         type: choice
         options:
-          - patch
-          - breaking-prerelease
+          - prerelease
+          - "8.0.0-beta.4"
```

Comment:
For some reason it tries to re-publish 8.0.0-beta.3 right now, if I use `prerelease`. After a new version tag is created (i.e. 8.0.0-beta.4), then `prerelease` properly bumps it up.